packages/documentation/src/stories/foundations/typography/list/list.snapshot.stories.ts

```diff
@@ -0,0 +1,132 @@
+import type { StoryObj } from '@storybook/web-components';
+import { schemes } from '@/shared/snapshots/schemes';
+import meta from './list.stories';
+import { html, TemplateResult } from 'lit';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+
+const { id, ...metaWithoutId } = meta;
+
+export default {
+  ...metaWithoutId,
+  title: 'Snapshots',
+};
+
+type Story = StoryObj;
+
+export const Lists: Story = {
+  render: () => {
+    return schemes(
+      () => html`
+        <h1>Lists</h1>
+
+        <h2>Bullet</h2>
+        <div class="row">
+          <div class="col-6">
+            <code>&lt;ul&gt;</code>
+            ${renderList({ el: 'ul' })} ${renderList({ el: 'ul', classList: 'list-revert' })}
+            ${renderList({ el: 'ul', classList: 'list-unstyled' })}
+          </div>
+
+          <div class="col-6">
+            <code>&lt;ul class="list-inline"&gt;</code>
+            ${renderList({
+              el: 'ul',
+              classList: 'list-inline',
+            })}
+            ${renderList({
+              el: 'ul',
+              classList: 'list-inline list-revert',
+            })}
+            ${renderList({
+              el: 'ul',
+              classList: 'list-inline list-unstyled',
+            })}
+          </div>
+
+          <div class="w-100 mt-32"></div>
+
+          <div class="col-6">
+            <code>&lt;ul class="list-bullet"&gt;</code>
+            ${renderList({ el: 'ul', classList: 'list-bullet' })}
+            ${renderList({ el: 'ul', classList: 'list-bullet list-revert' })}
+            ${renderList({ el: 'ul', classList: 'list-bullet list-unstyled' })}
+          </div>
+
+          <div class="col-6">
+            <code>&lt;ul class="list-bullet list-inline"&gt;</code>
```

Comment:
Shouldn't `list-bullet list-inline` have bullets? 
Also putting the lists on two columns make it hard to see the list-inline as there is not enough space for the list item to be on the same line.

Reply:
No, a `.list-bullet .list-inline` should not have any bullets.
Can you explain your second point again? I don't really get it.